### PR TITLE
Update certificate-authority-add-scep-overview.md

### PR DESCRIPTION
**Pull request by msbemba**
removed the Microsoft Graph API step as it is covered in 7.b Could find any reason to redo this again using the api

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by PRMerger17** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/2555

**Comment by Brenduns** [comment]
This PR is pending feedback from PM/Dev. Stay tuned.

File: memdocs/intune/protect/certificate-authority-add-scep-overview.md [hunk 1]
```diff
@@ -113,15 +113,7 @@ Be sure you have the required permissions to register an Azure AD app. See [Requ
...
-   1. Use *Microsoft Graph* to add the following permissions to the app:
-
-      - **Application.Read.All** (Read all applications).
-
-      For information on how to complete this step, see [Use Microsoft Graph to configure required Azure AD Graph permissions for an app registration](/graph/migrate-azure-ad-graph-configure-permissions).
-
-      > [!NOTE]  
-      > Previously, these permissions were configured by using Azure AD Graph, and available through the App registration UI. Azure AD Graph is now deprecated and will be retired on June 30, 2022. As part of this deprecation path, the capability to add Azure AD Graph permissions to the required permissions for an app registration through the Azure portal is now disabled.
-
+
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/certificate-authority-add-scep-overview.md:116]:
```suggestion
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by Brenduns** [comment]
Product Dev continues to investigate what the permissions that must be set via Graph are. This remains on hold until the correct procedure update is identified.

**Comment by Brenduns** [comment]
Still waiting for confirmation of permission from Dev. Remains on hold.

**Comment by Brenduns** [comment]
Check on 6/2 remains still pending...

**Comment by yogkumgit** [comment]
@Brenduns Any update.

**Comment by Brenduns** [comment]
@yogkumgit   Also bumped (again) with the Intune dev team.

**Comment by Brenduns** [comment]
@yogkumgit; @msbemba  I've just received the tested confirmation from the Dev team as to the permissions, which means I can finally accept this PR as the gospel for the content.  I regret that it took so long to get that validation, but with it we can finally have the approved/verified/endorsed content update.

#sign-off

**Pull request merged**